### PR TITLE
Fixes in Connections/MeshSync tables

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #
1. Search bug in meshsync
2. add column functionality
3. fix pagination
4. remove sort for metadata for sorting because that is not a column(bug)
5. some ui enhancements

image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/connections/MeshsyncTable.js [hunk 1]
```diff
@@ -373,6 +378,28 @@ export default function MeshSyncTable({ classes, updateProgress, search }) {
...
+        <div className={classes.createButton}>{/* <MesherySettingsEnvButtons /> */}</div>
+        <div
+          className={classes.searchAndView}
+          style={{
+            display: 'flex',
+            borderRadius: '0.5rem 0.5rem 0 0',
+          }}
+        >
+          <SearchBar
+            onSearch={(value) => {
```
Comment (Ckbhatia) [inline ui/components/connections/MeshsyncTable.js:391]: Why need an extra function?

File: ui/components/connections/MeshsyncTable.js [hunk 2]
```diff
@@ -373,6 +378,28 @@ export default function MeshSyncTable({ classes, updateProgress, search }) {
 
   return (
     <>
+      <div className={StyleClass.toolWrapper} style={{ marginBottom: '5px', marginTop: '-30px' }}>
+        <div className={classes.createButton}>{/* <MesherySettingsEnvButtons /> */}</div>
+        <div
+          className={classes.searchAndView}
+          style={{
```
Comment (Ckbhatia) [inline ui/components/connections/MeshsyncTable.js:385]: Should be moved to the class

File: ui/components/connections/MeshsyncTable.js [hunk 3]
```diff
@@ -46,18 +51,18 @@ export default function MeshSyncTable({ classes, updateProgress, search }) {
...
-          return (
-            <SortableTableCell
-              index={index}
-              columnData={column}
-              columnMeta={columnMeta}
-              onSort={() => sortColumn(index)}
-            />
-          );
-        },
+        // customHeadRender: function CustomHead({ index, ...column }, sortColumn, columnMeta) {
```
Comment (Ckbhatia) [inline ui/components/connections/MeshsyncTable.js:56]: Should be removed if not used

Comment (sudhanshutech) [reply ui/components/connections/MeshsyncTable.js:56]: removed for now, so if needed can be added again

**Pull request merged**